songpal/notification.py

```diff
@@ -155,7 +155,11 @@ class ContentChange(ChangeNotification, PlayInfo):
     @property
     def is_input(self):
         """Return if the change was related to input."""
-        return self.contentKind == "input"
+        return (
+            self.contentKind == "input"
+            if self.contentKind is not None
+            else self.kind == "input"
+        )
```

Comment:
I am assuming that both kind and contentKind can be used interchangeably? If so, I think it'd be a good idea check out how we could populate the already existing attribute using the second name. This would allow handling them consistently all across the board.

Reply:
Yes, it is. For the STR-AN1000 it switches between the two. Examples: `'kind': 'input'` for all `extInput`, but `'contentKind': 'radio'` for `radio:` source types, then other source types like `netService:audio` doesn't have the attribute at all.

Will look into populating the existing attribute.

Reply:
Have done it differently, but this is the limit of my python capabilities.

I looked at a solution like below. But it ended up being more code and not necessarily easy to understand what it does.
```
contentKind = attr.ib(default="self.kind")
```

Reply:
The `__attrs_post_init__` approach looks good to me!